tests/unit/Entity/ItemIdSetTest.php

```diff
@@ -16,6 +16,35 @@
  */
 class ItemIdSetTest extends \PHPUnit_Framework_TestCase {
 
+	public function testGetIterator() {
+		$set = new ItemIdSet();
+		$iterator = $set->getIterator();
+
+		$this->assertInstanceOf( 'Traversable', $iterator );
+	}
+
+	/**
+	 * @dataProvider serializationsProvider
+	 */
+	public function testGetSerializations( array $expected ) {
```

Comment:
What's the "less complex" alternative? Really, I was thinking about this but everything I can come up with is equally "complex" (probably depends on what "complex" means to you). Something like that?
```php
public function serializationsProvider() {
	return array(
		array( array( new ItemId( 'Q1' ) ), array( 'Q1' ) ),
	);
}
public function testGetSerializations( array $itemIds, array $serializations ) {
	$set = new ItemIdSet( $itemIds );
	$this->assertEquals( $serializations, $set->getSerializations() );
}
```